platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/main/java/com/swirlds/platform/test/simulated/config/ListBuilder.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2016-2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.platform.test.simulated.config;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A convenience class for building lists with repeating elements
+ */
+public class ListBuilder<T> {
+    private final List<T> list = new ArrayList<>();
+    private T lastElement = null;
+
+    private ListBuilder() {}
+
+    public static <T> ListBuilder<T> builder(final Class<T> type) {
```

Comment:
Null annotations and checks missing. I will stop adding that comment for future classes.
I would expect to have it in `src/main/java` in `platform-test`.

Reply:
i think i got them everywhere in new code in `swirlds-platform-test/src/main/java`